base/deprecated.jl

```diff
@@ -1355,6 +1362,15 @@ module DSP
 end
 using .DSP
 export conv, conv2, deconv, filt, filt!, xcorr
+@doc """
+DFT will be removed from Base. Use `AbstractFFTs` instead.
+""" -> DFT
+@doc """
+DSP will be removed from Base. Use the package with the same name instead.
+""" -> DSP
+@doc """
+FFTW will be removed from Base. Use the package with the same name instead.
+""" -> FFTW
```

Comment:
Should we bother documenting these moved modules?

Reply:
I have a test that all exported modules in Base are documented. Thats why I added these.

Reply:
Yea, I'm not sure we need a test like that. Adding a new exported module in Base does not happen very often so not sure its worth it. If you add a `## Base Modules` section in `doc/src/stdlib/base.md` (perhaps just after the `## Keywords` section) and list the new docstrings there, then CI will fail if these docstrings are lost.

Reply:
Yes I agree the test was useful for making sure that I did not forget any exported module, but is useless now. I will add the Modules to `base.md`.